### PR TITLE
📖 book: extend next steps of quick start

### LINKED ISSUE #8152: book create just one cluster
you say one or more k8s clusters but quck start has just one cluster 
and next question is what is  workload cluster  application is going to run?
image

### PR DESCRIPTION
**Pull request by Sajiyah-Salat**
**What this PR does / why we need it**:
in quick start there was info for first cluster but second cluster info was missing I have just added that. 
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #8152

### CODE REVIEW

File: docs/book/src/user/quick-start.md [hunk 1]
```diff
@@ -1567,8 +1567,11 @@ kind delete cluster
 
 ## Next steps
 
+Create a second workload cluster. Simply follow the steps outlined above, but remember to provide a different name for your second workload cluster.
+Deploying applications to your workload cluster - use the CNI deployment steps for pointers.<br>
 See the [clusterctl] documentation for more detail about clusterctl supported actions.
```
Comment (killianmuldoon) [inline docs/book/src/user/quick-start.md:1572]:
```suggestion
- Create a second workload cluster. Simply follow the steps outlined above, but remember to provide a different name for your second workload cluster.
- Deploy applications to your workload cluster. Use the CNI deployment steps for pointers.<br>
- See the [clusterctl] documentation for more detail about clusterctl supported actions.
```

Add bullet points to make this a little bit clearer.

File: docs/book/src/user/quick-start.md [hunk 2]
```diff
@@ -1567,8 +1567,11 @@ kind delete cluster
...
 
+
```
Comment (killianmuldoon) [inline docs/book/src/user/quick-start.md:1574]:
```suggestion
```

Remove line.

**Review by killianmuldoon — COMMENTED** [review]
Thanks for this!

Just a couple of nits to clear up.

File: docs/book/src/user/quick-start.md [hunk 3]
```diff
@@ -1567,9 +1567,9 @@ kind delete cluster
 
 ## Next steps
 
-Create a second workload cluster. Simply follow the steps outlined above, but remember to provide a different name for your second workload cluster.
-Deploying applications to your workload cluster - use the CNI deployment steps for pointers.<br>
-See the [clusterctl] documentation for more detail about clusterctl supported actions.
+- Create a second workload cluster. Simply follow the steps outlined above, but remember to provide a different name for your second workload cluster.
+- Deploy applications to your workload cluster. Use the CNI deployment steps for pointers.<br>
```
Comment (killianmuldoon) [inline docs/book/src/user/quick-start.md:1571]:
```suggestion
- Deploy applications to your workload cluster. Use the [CNI deployment steps](#deploy-a-cni-solution) for pointers.
```

**Review by killianmuldoon — APPROVED** [review]
/lgtm

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

@Sajiyah-Salat could you please squash commits?

**Comment by oscr** [comment]
> @Sajiyah-Salat could you please squash commits?

@Sajiyah-Salat Here is a description (how to squash commits)[https://github.com/kubernetes-sigs/cluster-api/blob/main/CONTRIBUTING.md#contributing-a-patch] from our contributing guide.

**Comment by Sajiyah-Salat** [comment]
I am stuck here. I have squashed my commits. but I dont know from where this other commint came.

**Comment by killianmuldoon** [comment]
Hi @Sajiyah-Salat - looks like you're running into some git stickyness :sweat_smile: - happens to us all. The following steps should help you get out of it:

```
git checkout sajiyah-#8152

git reset --hard main

git cherry-pick 87b605152c1a6e470c8fe97417ed284d94c7203a

git push -f NAME_OF_BRANCH
```

This should reset your branch to look exactly like main, cherry-pick the commit that contains the relevant changes and then push those changes here.

**Comment by Sajiyah-Salat** [comment]
I got this error while following the above command

````
git push -f sajiyah-#8152
fatal: 'sajiyah-#8152' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by oscr** [comment]
> I got this error while following the above command
> 
> ```
> git push -f sajiyah-#8152
> fatal: 'sajiyah-#8152' does not appear to be a git repository
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.
> ```

`git push` takes the repo as first argument. So depending on your workflow you either want to push to origin or your forks name (`git remote` will show you the name of your fork). I have my fork named "fork" as an origin so for me it's `git push fork -f`.

**Comment by Sajiyah-Salat** [comment]
ok, I got it. But now there are two commits with same name. what should I do? Also, Thank you for teaching me the squashing part. It always frustrates me out.

**Comment by oscr** [comment]
Looks like to separate commits with the same content. You should be able to solve it by squashing them: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

As a side note for future prs: I avoid getting multiple commits by using `git commit --amend` and then `git push <my repo> -f`.

**Comment by killianmuldoon** [comment]
@Sajiyah-Salat  - if you're stuck with the rebase process you can start again from main, add your changes to the quick-start and then force-push the changes to this branch. That will give you a full reset with only the changes you want.

**Comment by Sajiyah-Salat** [comment]
I will open a new pr. and add changes there with single commit.

**Comment by oscr** [comment]
Great work! 

/lgtm

**Pull request merged**